chapters/hub/hub-api.md

```diff
@@ -0,0 +1,67 @@
+# Hub API
+
+Jina Hub exposes several handy commands, namely `list`, `build` and `push` with configurable parameters. These APIs constitute the user’s experience using `jina cli` for a seamless experience with the Hub.
+
+
+## `jina hub` installation:
```

Comment:
Change to `## Install Jina Hub` or maybe just `## Install`

**Jina Hub** not `jina hub` because we're talking about installing the Jina Hub project, not using the `jina hub` CLI command

@FionnD do we have a standard for this? I've seen:
- `Installing Jina`
- `Install Jina`
- `Set up Jina`
- `Setup Jina`
- `Set-up Jina`
- `Jina Installation`
- `Jina setup` (plus variants of word `setup` like above)

Reply:
No we don't have a standard. I'll add it to my notes to improve the style guide.